packages/core/useTitle/index.md

```diff
@@ -7,7 +7,7 @@ category: Browser
 Reactive document title.
 
 ::: tip
-When using with Nuxt 3, this function will **NOT** be auto imported in favor of Nuxt's built-in `useTitle()`.
+This composable isn't compatible with SSR. When using with Nuxt 3, this function will **NOT** be auto imported in favor of Nuxt's built-in composables such as `useHead()` and `useSeoMeta()`.
```

Comment:
>  When using with Nuxt 3, this function will **NOT** be auto imported in favor of Nuxt's built-in composables such as `useHead()` and `useSeoMeta()`.

I think we hsould remove this. `useTitle` allows to retrieve the current title in any component.
`useHead` and `useSeoMeta` does not allow it. While it returns the `ActiveHead` instance, it does not return to the final and resolved value of the title in the head.

Reply:
Do you suggest to add auto import for this composable in Nuxt?
Can you please explain what you mean it can be used in any component, but useHead not?

Reply:
We can add it back. But we need to tell users that it is not SSR compatible in the JSDoc.

If you call `useHead({ title: 'hello' })`, it will return an `ActiveHeadEntry`. This entry is not injected into child components. 
Even if you provide this entry, this modification doesn't not reflect the final state your `Head` instance because other components can also call `useHead` -> this entry only contains the modification of the `useHead` call.

You could call `getActiveHead` from `unhead` but that also wouldn't work because `unhead` resolves asynchronously all tags and you would only get the state of the `Head` when the component is being mounted, not when everything has been rendered in SSR.

Reply:
Thank you for the wide explanation, @huang-julien! I got the difference :)
I updated the documentation, added a description to JSDoc, and removed the disabled auto import in Nuxt for `useTitle`.